imap/caldav_alarm.c

```diff
@@ -909,7 +978,13 @@ EXPORTED int caldav_alarm_sync_nextcheck(struct mailbox *mailbox, const struct i
 /* delete all alarms matching the event */
 HIDDEN int caldav_alarm_delete_record(const char *mboxname, uint32_t imap_uid)
 {
-    return update_alarmdb(mboxname, imap_uid, 0);
+    return update_alarmdb(mboxname, imap_uid, 0, 0, 0);
+}
+
+static int caldav_alarm_bump_nextcheck(const char *mboxname,
+                                       uint32_t imap_uid, time_t nextcheck)
+{
+    return update_alarmdb(mboxname, imap_uid, nextcheck, 0, 0);
```

Comment:
Should this be using a constant for `atype`?

Reply:
Ahh, I have followed up - it's OK :)  If `atype` is zero, then we do an update which doesn't change the type.  Fair enough.